README.md

```diff
@@ -1,2 +1,126 @@
-# sample_alert_info
-Release notes for sample Rubin Observatory alert packets
+# Rubin Observatory Sample Alerts
+
+The Vera C. Rubin Observatory is releasing sample alert packets produced by its pipelines run on precursor survey data.
+These alerts are intended to provide candidate community alert brokers as well as science users the chance to preview the alert format and contents and to provide feedback to the project.
+
+## Obtaining the Data
+
+Sample alert data may be obtained at [this link](https://lsst.ncsa.illinois.edu/~ebellm/sample_precursor_alerts/).
+We anticipate providing reprocessed and updated alert samples as pipeline development progresses, so the data are arranged in a directory structure:
+
+```
+    sample_precursor_alerts/
+	YYYY-MM-DD/
+	    dataset/
+		all_visits_{dataset}_{date}.avro
+		association_{dataset}_{date}.db
+		single_ccd_sample_{dataset}_{date}.avro
+		single_visit_sample_{dataset}_{date}.avro
+```
+
+### Alert packets
+
+Alerts are provided as Snappy-compressed [Avro](https://avro.apache.org/) files.
+The schema used to store the alerts is included in the file.
+
+As the complete sample is about 1 TB, We include two smaller subsets--alerts from a single CCD-visit, and all of the alerts from all CCDs in a single visit.
+
+### Database
+In addition to accessing alert packets via community brokers, during the LSST survey science users with data rights will be able to query the relational Prompt Products Database (PPDB).  
+The contents of the alerts are taken directly from the PPDB `DIASource`, `DIAForcedSource`, `DIAObject`, and `SSObject` records.
+For some use cases direct queries to the PPDB may be more efficient than processing alert packets.
+
+To allow users to explore these different modes of data access, we are also providing the PPDB (`association_{dataset}_{date}.db`) from which the sample alerts were generated as an [SQLite](https://www.sqlite.org) database.
+The SQLite database can be queried using SQL through a variety of interfaces.
+
+Note that in operations, users will query the PPDB using [ADQL](http://www.ivoa.net/documents/ADQL/) via a [TAP service](http://www.ivoa.net/documents/TAP/).
+
+### Software
+Users should install [lsst.alert.packet](https://github.com/lsst/alert_packet) to parse alerts and optionally [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) and [alert-stream](https://github.com/lsst-dm/alert_stream) if they wish to simulate a Kafka alert stream.
+
+## External References and Additional Resources
+
+The [Data Products Definition Document (DPDD)](http://ls.st/dpdd) provides a change-controlled guide to what scientific contents are planned for alerts and other Rubin Observatory data products.
```

Comment:
Does the latest DPDD include the DiaForcedSource table?

Reply:
Yes (Table 2)